packages/schematics/angular/application/index.ts

```diff
@@ -155,10 +155,10 @@ function addAppToWorkspaceFile(options: ApplicationOptions, appDir: string): Rul
     || options.inlineStyle === true
```

Comment:
Shouldn't the minimal check be added here as well?

Can you add a Unit tests please?

Reply:
Yes, sorry for overlooking this. Otherwise it doesn't make sense to check for `minimal` inside the nested if statements.